demosR/phylogeny/dummyData.R

```diff
@@ -0,0 +1,248 @@
+dummyNodes <- list(
```

Comment:
Do the contents of `/demosR/phylogeny` serve any purpose?

I can't find any reason for them to be there, they don't affect the examples as far as I can tell. 

 :hocho:

Reply:
good catch -- I removed that demo (for now) because it was unfinished, will clean up this directory also.